test/org/freedesktop/gstreamer/SampleTester.java

```diff
@@ -76,7 +76,7 @@ public FlowReturn newSample(AppSink appSink) {
 
     private SampleTester(Consumer<Sample> callback) {
         ArrayList<GError> errors = new ArrayList<GError>();
-        String pipeline_descr = "videotestsrc ! videoconvert ! appsink name=myappsink";
+        String pipeline_descr = "videotestsrc is-live=true ! videoconvert ! appsink name=myappsink";
```

Comment:
Please don't change the default pipeline unless there's a valid reason?  If you need a different pipeline for a particular test, you can pass it in to the other methods on SampleTester.